curriculum/challenges/english/10-coding-interview-prep/rosetta-code/discordian-date.md

```diff
@@ -7,16 +7,15 @@ dashedName: discordian-date
 ---
 
 # --description--
+The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. The week here consists of seven days, whereas there are 52 weeks and 1 day per year. The year consists of 365 days, with a leap day added to February every four years, which is called a leap year (except for three centurial years for every 400 years).
```

Comment:
```suggestion
The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. The week here consists of seven days, whereas there are 52 weeks and 1 day per year. The year consists of 365 days, with a leap day added to February in leap years (these are every 4 years, in years divisible by 4, but if the year ends with 00 it is a leap year only if it's divisible by 400).
```

Reply:
Can we rework to avoid the parenthetical statement? I.e. pull that text out into sentences outside of the ()

Reply:
?
```suggestion
The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. The week here consists of seven days, whereas there are 52 weeks and 1 day per year. The year consists of 365 days, with an extra day added to February in leap years.

Leap years happens every 4 years, in years divisible by 4, but if the year ends with 00 it is a leap year only if it's divisible by 400
```